jicofo/rootfs/defaults/jicofo.conf

```
@@ -187,6 +188,12 @@ jicofo {
       enable-multi-stream-backward-compat = {{ .Env.JICOFO_MULTI_STREAM_BACKWARD_COMPAT | toBool }}
       {{ end }}
 
+      {{ if .Env.ENABLE_SHARED_DOCUMENT_RANDOM_NAME }}
```

```suggestion
      {{ if $ENABLE_SHARED_DOCUMENT_RANDOM_NAME }}
```
Since we have a variable defined we ought to use it.  Otherwise looks great to me!